website/content/en/highlights/2022-02-08-0-20-0-upgrade-guide.md

```diff
@@ -15,6 +15,7 @@ Vector's 0.20.0 release includes **breaking changes**:
 1. [Change to set expiration behavior in `prometheus_exporter` sink](#prom-exporter-set-expiration)
 1. [New metrics behavior for `route` transform](#metrics-route-transform)
 1. [New internal events for `loki` sink](#events-loki-sink)
+1. [If statement predicates in `VRL` need to handle errors](#vrl-fallible-predicates)
```

Comment:
```suggestion
1. [`if` statement predicates in `VRL` need to handle errors](#vrl-fallible-predicates)
```

Could otherwise trip up someone reading that grammatically as an if 😄